agent/resources/resources_linux.go

```diff
@@ -93,6 +113,13 @@ func (c *cgroupWrapper) setupCgroup(task *api.Task) error {
 		return errors.Wrapf(err, "resource: setup cgroup: unable to create cgroup at %s for task: %s", cgroupRoot, task.Arn)
 	}
 
+	// echo 1 > memory.use_hierarchy
+	memoryHierarchyPath := filepath.Join(c.cgroupPath, memorySubsystem, cgroupRoot, memoryUseHierarchy)
+	err = c.ioutil.WriteFile(memoryHierarchyPath, []byte(strconv.Itoa(1)), os.FileMode(0))
```

Comment:
these are still magic numbers. you should be able to const both of these with descriptive names. sorry for missing this in the last round.

Reply:
ah it's fine. i made this change earlier and i think it got lost while trying to revert back to per task `memory.use_hierarchy`